pyvista/plotting/renderer.py

```diff
@@ -874,7 +874,7 @@ def add_actor(
         self.AddActor(actor)  # must add actor before resetting camera
         self._actors[name] = actor
 
-        if reset_camera or not self.camera_set and reset_camera is None and not rv:
+        if reset_camera or (not self.camera_set and reset_camera is None and not rv):
```

Comment:
This needs careful review, I don't think the assumption here by ruff is correct

Reply:
While ruff has not changed the python logic, it highlights the underlying issue.  The correct logic here is dense and not immediately obvious to me.  Could be a bug here....

Reply:
`reset_camera` is declared to be of type `bool`, so I'm not sure when it can be `None`.